### PR TITLE
Fix empty image selection error for file-image interface

### PR DESCRIPTION
**Pull request by licitdev**
Fix image selection error for file-image interface when selection is empty.

### Before

https://github.com/directus/directus/assets/26413686/0ba13542-6dd5-4c71-8bf4-55eb62c572f1

### After

https://github.com/directus/directus/assets/26413686/265fe8a3-568f-4470-8f58-e9770048ddc6

### CODE REVIEW

**Review by paescuj — APPROVED** [review]
👍

**Review by br41nslug — APPROVED** [review]
LGTM 🚀

File: app/src/interfaces/file-image/file-image.vue [hunk 1]
```diff
@@ -185,6 +185,10 @@ async function imageErrorHandler() {
 	}
 }
 
+function onUpload(image: any) {
+	if (image) update(image.id);
```
Comment (br41nslug) [inline app/src/interfaces/file-image/file-image.vue:189]:
```suggestion
	if (image?.id) update(image.id);
```
To be on the safe side should we check for the actual ID? 🤔

Comment (rijkvanzanten) [reply app/src/interfaces/file-image/file-image.vue]: In this case `image` is typed as `any`, so we have to assume that it might be.. `any`thing 🙂

**Review by rijkvanzanten — APPROVED** [review]

**Pull request merged**